src/ResultUtils/OptionCE.fs

```diff
@@ -0,0 +1,70 @@
+namespace ResultUtils
+
+open System
+
+[<AutoOpen>]
+module OptionCE =
```

Comment:
is this the same we have in geewallet?

Reply:
Not source-wise identical. But yes, it's the same thing.

Reply:
what's the difference?

Reply:
It was copy-pasted from the `ResultCE` module in DNL (then modified), rather than copy-pasted from the geewallet code. Other than that, it supports `try`/`using`/`for` and `while` blocks in `option { .. }` blocks whereas the geewallet one doesn't. Also it looks like the geewallet version has a bug: `Zero` should return `Some ()` rather than `None`.